specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -792,20 +859,20 @@ paths:
       parameters:
         - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/parameters/fields"
         - in: query
-          name: burgerservicenummer
+          name: ingeschrevenpersoon__burgerservicenummer
           description: "Het burgerservicenummer van de ingeschreven persoon."
           required: false
           schema:
             type: string
             maxLength: 9
         - in: query
-          name: kadasterpersoonidentificatie
+          name: persoon__kadasterpersoonidentificatie
           description: "De kadastrale identificatie van een persoon of een rechtspersoon die bij het kadaster geregistreerd is."
           required: false
           schema:
             type: string
         - in: query
-          name: typepersoon
+          name: persoon__typepersoon
```

Comment:
Is dit niet beslaglegger__type? Kan je dan ook de description aanpassen, want die helpt hier ook niet duidelijk te maken wat voor persoon dit is.

Reply:
kan je het schema hiervan ook aanpassen, zodat het een enum wordt, zoals in PersoonType_enum is gedefinieerd?

Reply:
Persoontype_enum staat in het schema.

Beslaglegger is helemaal geen resource. Dat is een rol die een persoonheeft in een beslag. Daarom vindt ik hier persoon__typepersoon wel goed.